pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -191,7 +191,7 @@ func (pl *DefaultPreemption) FindCandidates(ctx context.Context, state *framewor
 		return nil, err
 	}
 	if len(allNodes) == 0 {
-		return nil, core.ErrNoNodesAvailable
+		return nil, pluginshelper.ErrNoNodesAvailable
```

Comment:
we don't need to return the same error here, so please keep the error definition of `ErrNoNodesAvailable` where it was, and change this to `return nil, fmt.Errorf("no nodes available to schedule pods")`

Reply:
For extra context:
If there are no nodes available, PostFilter is not called https://github.com/kubernetes/kubernetes/blob/8515b2a97b31798efea566c6edd00fe2668e8907/pkg/scheduler/scheduler.go#L481

Reply:
`FindCandidates` is only called in PostFilter, So it should be fine to not use ErrNoNodesAvailable.